ember-client/shared/src/main/scala/org/http4s/ember/client/internal/ClientHelpers.scala

```diff
@@ -295,4 +292,11 @@ private[client] object ClientHelpers {
         case _ => false
       }
   }
+
+  private case class MissingHost() extends RuntimeException("Invalid hostname")
+
+  private case class MissingPort() extends RuntimeException("Invalid port")
```

Comment:
Should we include the actual value, to help with debugging the error?

Reply:
I thought about that, but left it without actual values like Christopher has done in `H2Client`.

Reply:
Ok, fair enough. We can always revisit.